tests: some driver tests of frdm_k64f build failed in twister (shows devicetree error)

**Describe the bug**
After SDHC support was added into commit: 33b851f71e4268685e850f053d7577026f052b53, build tests by twister will fail due to 
```
devicetree error: 'label' is marked as deprecated in 'properties:' in /home/xx/zephyrproject/zephyr/dts/bindings/sdhc/zephyr,sdhc-spi-slot.yaml for node /soc/spi@4002d000/sdhc@0."
```

We simply remove the label of shdc in dts file can avoid this, but we are not sure if this is right solution.

```
diff --git a/boards/arm/frdm_k64f/frdm_k64f.dts b/boards/arm/frdm_k64f/frdm_k64f.dts
index 770225e851..c6844cada2 100644
--- a/boards/arm/frdm_k64f/frdm_k64f.dts
+++ b/boards/arm/frdm_k64f/frdm_k64f.dts
@@ -156,7 +156,6 @@ arduino_spi: &spi0 {
                compatible = "zephyr,sdhc-spi-slot";
                reg = <0>;
                status = "okay";
-               label = "SDHC_0";
                mmc {
                        compatible = "zephyr,sdmmc-disk";
                        status = "okay";
```

**To Reproduce**
Steps to reproduce the behavior:
1. source zephyr-env.sh
2. twister -T tests/driver/dac/dac_api -vv
3. See build error

**Expected behavior**
Test can build and run by west and twister.

**Impact**
This impact some tests in frdm_k64f. Build error happens when we build/run it by twister. This block some PRs, Ex,
https://github.com/zephyrproject-rtos/zephyr/pull/49826
https://github.com/zephyrproject-rtos/zephyr/pull/50020

**Logs and console output**
```
Loading Zephyr default modules (Zephyr base).
-- Application: /home/mm/zephyrproject/zephyr/tests/drivers/dac/dac_api
-- Found Python3: /usr/bin/python3.8 (found suitable exact version "3.8.10") found components: Interpreter
-- Cache files will be written to: /home/mm/.cache/zephyr
-- Zephyr version: 3.1.99 (/home/mm/zephyrproject/zephyr)
-- Found west (found suitable version "0.12.0", minimum required is "0.7.1")
-- Board: frdm_k64f
-- Found host-tools: zephyr 0.15.0 (/home/mm/zephyr-sdk-0.15.0)
-- Found toolchain: zephyr 0.15.0 (/home/mm/zephyr-sdk-0.15.0)
-- Found Dtc: /home/mm/zephyr-sdk-0.15.0/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.6.0", minimum required is "1.4.6")
-- Found BOARD.dts: /home/mm/zephyrproject/zephyr/boards/arm/frdm_k64f/frdm_k64f.dts
devicetree error: 'label' is marked as deprecated in 'properties:' in /home/mm/zephyrproject/zephyr/dts/bindings/sdhc/zephyr,sdhc-spi-slot.yaml for node /soc/spi@4002d000/sdhc@0.
-- In: /home/mm/zephyrproject/zephyr/twister-out/frdm_k64f/tests/drivers/dac/dac_api/drivers.dac.api/zephyr, command: /usr/bin/python3.8;/home/mm/zephyrproject/zephyr/scripts/dts/gen_defines.py;--dts;/home/mm/zephyrproject/zephyr/twister-out/frdm_k64f/tests/drivers/dac/dac_api/drivers.dac.api/zephyr/zephyr.dts.pre;--dtc-flags;'';--bindings-dirs;/home/mm/zephyrproject/zephyr/dts/bindings;--header-out;/home/mm/zephyrproject/zephyr/twister-out/frdm_k64f/tests/drivers/dac/dac_api/drivers.dac.api/zephyr/include/generated/devicetree_generated.h.new;--dts-out;/home/mm/zephyrproject/zephyr/twister-out/frdm_k64f/tests/drivers/dac/dac_api/drivers.dac.api/zephyr/zephyr.dts.new;--edt-pickle-out;/home/mm/zephyrproject/zephyr/twister-out/frdm_k64f/tests/drivers/dac/dac_api/drivers.dac.api/zephyr/edt.pickle;--edtlib-Werror;--vendor-prefixes;/home/mm/zephyrproject/zephyr/dts/bindings/vendor-prefixes.txt
CMake Error at /home/mm/zephyrproject/zephyr/cmake/modules/dts.cmake:230 (message):
  gen_defines.py failed with return code: 1
Call Stack (most recent call first):
  /home/mm/zephyrproject/zephyr/cmake/modules/zephyr_default.cmake:108 (include)
  /home/mm/zephyrproject/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:66 (include)
  /home/mm/zephyrproject/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:92 (include_boilerplate)
  CMakeLists.txt:5 (find_package)


-- Configuring incomplete, errors occurred!

```

**Environment (please complete the following information):**
 - OS: (e.g. Linux, MacOS, Windows)
 - Toolchain Zephyr SDK 0.15.0
 - Commit SHA: 33b851f71e4268685e850f053d7577026f052b53

--- a/boards/arm/frdm_k64f/frdm_k64f.dts
+++ b/boards/arm/frdm_k64f/frdm_k64f.dts
@@ -156,11 +156,9 @@ arduino_spi: &spi0 {
 		compatible = "zephyr,sdhc-spi-slot";
 		reg = <0>;
 		status = "okay";
-		label = "SDHC_0";
 		mmc {
 			compatible = "zephyr,sdmmc-disk";
 			status = "okay";
-			label = "SDMMC_0";
 		};
 		spi-max-frequency = <24000000>;
 	};

